### PR TITLE
Fix undefined method current alchemy user

### PR DESCRIPTION
**Pull request by mickenorlen**
A fix for:
https://github.com/AlchemyCMS/alchemy_cms/issues/1715
https://github.com/AlchemyCMS/alchemy_cms/issues/1620

Tested in this repo:
https://github.com/mbajur/alchemy-exception

As mentioned here:
https://github.com/AlchemyCMS/alchemy_cms/issues/1715#issuecomment-587160284

### CODE REVIEW

**Comment by mbajur** [comment]
Awesome! I was planning to resolve that in the exact same way you did but i decided it's impossible alchemy_cms works for everyone but me without these lines so it must be something else...

Thank you!

**Comment by mickenorlen** [comment]
No problem! 

Yeah I don't know if it's the right solution either, but at least mb it can help them identify the problem :P

File: app/controllers/alchemy/admin/user_sessions_controller.rb [hunk 1]
```diff
@@ -1,6 +1,11 @@
 module Alchemy
   module Admin
     class UserSessionsController < ::Devise::SessionsController
+      include Alchemy::ConfigurationMethods
+      include Alchemy::AbilityHelper
+      include Alchemy::ControllerActions
```
Comment (tvdeyen) [inline app/controllers/alchemy/admin/user_sessions_controller.rb]: The only module we need is the `Alchemy::ControllerActions` as this provides the `current_alchemy_user` method. All other modules should not be included.

File: app/controllers/alchemy/admin/user_sessions_controller.rb [hunk 2]
```diff
@@ -12,6 +17,7 @@ class UserSessionsController < ::Devise::SessionsController
       before_action :check_user_count, :only => :new
 
       helper 'Alchemy::Admin::Base'
+      helper 'Alchemy::Admin::Form'
```
Comment (tvdeyen) [inline app/controllers/alchemy/admin/user_sessions_controller.rb]: Why is this necessary?

File: app/controllers/alchemy/admin/user_sessions_controller.rb (hunk 1 above)
Comment (tvdeyen) [reply app/controllers/alchemy/admin/user_sessions_controller.rb]: Actually this module is already included in `Alchemy::BaseController` and the `Devise::SessionsController` inherits from that. I wonder why the method is not available 🤔 It should be.

File: app/controllers/alchemy/admin/user_sessions_controller.rb (hunk 2 above)
Comment (mickenorlen) [reply app/controllers/alchemy/admin/user_sessions_controller.rb]: I just tested it in the repo and all those includes were necessary for one reason or another.
The form helpers for rendering the re-login views with `alchemy_form_for`

Where does `Devise::SessionsController` inherit from  `Alchemy::BaseController` ?

**Comment by tvdeyen** [comment]
The issue has been closed https://github.com/AlchemyCMS/alchemy_cms/issues/1715#issuecomment-590820880

The gem was not installed correctly. It is necessary to run the install task so that Devise gets configured correctly.
